src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_AutoSuggestBox.cs

```diff
@@ -230,6 +230,51 @@ public async Task When_Submitting_After_Typing_Text()
 			SUT.Focus(FocusState.Programmatic);
 			await WindowHelper.WaitForIdle();
 		}
+
+		[TestMethod]
+		public async Task When_Using_Custom_ItemContainerStyle()
+		{
+			AutoSuggestBox SUT = new AutoSuggestBox
+			{
+				ItemContainerStyle = new Style(typeof(ListViewItem))
+				{
+					Setters =
+								{
+									new Setter(Control.HorizontalAlignmentProperty, HorizontalAlignment.Stretch),
+								},
+				}
+			};
+			string[] suggestions = { "a1", "a2", "b1", "b2" };
+			bool eventRaised = false;
+			SUT.TextChanged += (s, e) =>
+			{
+				if (e.Reason == AutoSuggestionBoxTextChangeReason.UserInput)
+				{
+					s.ItemsSource = suggestions.Where(i => i.StartsWith(s.Text));
+					eventRaised = true;
+				}
+			};
+			WindowHelper.WindowContent = SUT;
+			await WindowHelper.WaitForIdle();
+			var textBox = (TextBox)SUT.GetTemplateChild("TextBox");
+			var listView = (ListView)SUT.GetTemplateChild("SuggestionsList");
+			SUT.Focus(FocusState.Programmatic);
+
+			eventRaised = false;
+			textBox.ProcessTextInput("a");
+			await WindowHelper.WaitForIdle();
+			await WindowHelper.WaitFor(() => eventRaised);
+			Assert.AreEqual(2, listView.ItemsPanelRoot.Children.Count);
```

Comment:
listView.ItemsPanelRoot.Children doesn't work on Android? Or other problem?

Reply:
I tested on Android and found listView.ItemsPanelRoot.Children.Count is 0(zero).